tests/integration/linux_bridge_test.py

```diff
@@ -295,6 +297,19 @@ def test_linux_bridge_over_bond_over_slave_in_one_transaction(bond0):
     assertlib.assert_absent(bridge_name)
 
 
+def test_rollback_for_linux_bridge():
+    original_state = libnmstate.show()
+    bridge_name = TEST_BRIDGE0
+    bridge_state = _create_bridge_subtree_config(("invalid_port",))
```

Comment:
The invalid port name is confusing and may fail the transaction before anything was created (which will not check what you intended to check).
Please add a key-value that is invalid at the `desired_state` end result, just before you apply.

BTW: please always use single quotes (`).

Reply:
Done